eng/versioning/external_dependencies.txt

```diff
@@ -166,9 +166,9 @@ com.microsoft.azure:azure-storage;8.0.0
 com.microsoft.azure:msal4j;1.8.1
 com.microsoft.azure:msal4j-persistence-extension;1.0.0
 com.sun.activation:jakarta.activation;1.2.2
-io.opentelemetry:opentelemetry-api;0.14.1
-io.opentelemetry:opentelemetry-sdk;0.14.1
-io.opentelemetry:opentelemetry-semconv;0.14.1
+io.opentelemetry:opentelemetry-api;0.16.0
+io.opentelemetry:opentelemetry-sdk;0.16.0
+com.google.errorprone:error_prone_annotations;2.5.1
```

Comment:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2188

Reply:
Looks like this was fixed 3 days ago in their main branch: https://github.com/open-telemetry/opentelemetry-java/pull/2898/files

Reply:
Yes, I think this will make it in their next release.